common-content/en/module/tools/wc/index.md

```diff
@@ -0,0 +1,19 @@
++++
+title = "wc"
+headless = true
+time = 10
+facilitation = false
+emoji= "💻"
+[objectives]
+    1="Count the bytes, words, and lines of a file with wc"
+    2="Count just the words of a file with wc"
++++
+
+`wc` is a tool for counting the bytes, words, or lines in a file.
+
+Learn about it from its man page (and the backlog exercises).
+
+Some commonly used flags you should understand:
```

Comment:
Set them an exercise to check their understanding? Count the words in this block using wc?